pkg/controller/controlplanecerts/controlplanecerts_controller.go

```diff
@@ -297,7 +297,19 @@ func (r *ReconcileControlPlaneCerts) generateControlPlaneCertsSyncSet(cd *hivev1
 		cdLog.WithError(err).Error("cannot add typemeta to syncset resources")
 		return nil, err
 	}
+	// kubeAPIServerPatch indicates sets the forceRedeploymentField on the kube API server cluster operator
+	// to a hash of all the cert secrets. If the content of the certs secrets changes, then the new
+	// hash value will force the kube API server to redeploy and apply the new certs.
```

Comment:
The in-cluster operator watches this annotation, and if it changes will automatically redeploy, but do nothing if the value stays steady?

Reply:
Yes